app/views/organisations/_separate_website.html.erb

```diff
@@ -1,25 +1,26 @@
-<%= render "govuk_publishing_components/components/heading", {
-    text: organisation.title,
-    heading_level: 2
-} %>
+<% content_for :breadcrumbs do %>
+  <div class="organisations__margin-bottom">
+    <h1 class="visually-hidden"><%= @header.org.title %></h1>
+  </div>
+<% end %>
 
-<div class="separate-website-label">
-  <%= "#{organisation.title} has a" %> <%= link_to "separate website", "#" %>
+<div class="grid-row">
+  <div class="column-one-third organisations__margin-bottom">
+    <%= render "govuk_publishing_components/components/organisation_logo", {
+      organisation: @header.organisation_logo
+    } %>
+  </div>
 </div>
 
-<div class="organisation-body">
-  <%= organisation.body %>
-</div>
+<%= render "govuk_publishing_components/components/notice", {
+  title: "#{organisation.title} has a <a href='/'>separate website</a>".html_safe
```

Comment:
We need to actually link to the organisation's website here.. although I tried can't seem to find that info in the content item (on live) so I'm not sure how Whitehall does this currently.